assets/i18n/id.json

```diff
@@ -25,245 +24,238 @@
+    "dgc-browser-modal-dogechain": "Domet DogeChain", 
     "dgc-browser-modal-holytransaction": "HolyTransaction", 
     "dgc-browser-modal-blockio": "Block.io", 
-    "dgc-browser-modal-online-disclaimer": "<strong>Catatan:</strong> Dompet online ini tidak dimiliki atau pun dikelola oleh Dogecoin. Meskipun menurut kami entitas dibelakang mereka terpercaya tetapi tetap digunakan dengan resiko sendiri.", 
+    "dgc-browser-modal-online-disclaimer": "<strong>Catatan:</strong> Dompet daring berikut tidak dimiliki dan dikelola oleh Dogecoin. Sejauh ini orang-orang di baliknya si terpercaya - tapi resiko ditanggung sendiri.", 
     "dgc-ios-wallet-modal-important": "PENTING:", 
-    "dgc-ios-wallet-modal-view-only": "MyDoge untuk iOS hanya dapat digunakan untuk melihat dompet.", 
-    "dgc-ios-wallet-modal-not-send": "MyDoge <strong>tidak dapat</strong> mengirim Dogecoins. Kamu harus menyiapkan dompet Dogeoin yang sebenarnya sebelum kamu dapat menggunakan penampil dompet Dogecoin.", 
-    "dgc-ios-wallet-modal-understand": "Saya Mengerti, Ayo!", 
-    "dgc-ios-wallet-modal-los-gehts": "Klik tombol untuk memproses ke halaman Appstore.", 
+    "dgc-ios-wallet-modal-view-only": "MyDoge untuk iOS hanya dapat digunakan untuk melihat isi dompet.", 
+    "dgc-ios-wallet-modal-not-send": "MyDoge <strong>tidak bisa</strong> mengirim Dogecoin. Kamu harus membuat dompet Dogecoin beneran sebelum bisa menggunakan aplikasi untuk melihat dompet Dogecoin ini.", 
+    "dgc-ios-wallet-modal-understand": "Oke Paham, Lanjut!", 
+    "dgc-ios-wallet-modal-los-gehts": "Klik tombolnya untuk melanjutkan ke halaman Appstore.", 
     "dgc-learnmore-modal-title": "PELAJARI LEBIH LANJUT:", 
-    "dgc-learnmore-modal-dogeisfun": "Dogecoin adalah mata uang/aset digital yang menyenangkan, baru, dan berkembang pesat.", 
-    "dgc-learnmore-modal-crypto": "Bentuk dari Dogecoin disebut \"kripto kurensi\"; sebuah aset digital atau mata uang digital.", 
-    "dgc-learnmore-modal-anonymous": "Kripto kurensi sepenuhnya anonim, terdesentralisasi, dan secara ekstrim aman.", 
-    "dgc-learnmore-modal-wallet": "Dogecoin digunakan dengan sebuah dompet di komputermu, ponsel pintarmu, atau di sebuah website.",
-    "dgc-learnmore-modal-uses": "Kamu dapat menggunakan itu untuk membeli barang - barang dan servis, atau diperjual-belikan untuk aset lainnya (kripto kurensi lainnya atau mata uang tradisional seperti dolar Amerika (dolar US)).", 
-    "dgc-learnmore-modal-tipping": "Salah satu kegunaan Dogecoin yang populer adalah \"memberi tip\" kepada sesama pengguna internet yang membuat atau membagikan konten hebat. Tip tersebut dapat dianggap sebagai tanda suka dengan nilai yang nyata yang bisa digunakan di seluruh internet.", 
+    "dgc-learnmore-modal-dogeisfun": "Dogecoin adalah bentuk mata uang digital yang asik, baru dan tumbuh dengan cepat.", 
```

Comment:
I think we can use "merupakan" instead of "adalah" since our intention is to describe how Dogecoin looks like